.github/workflows/ci.yml

```
@@ -103,8 +103,8 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10"]
-        nautobot-version: ["1.4.10", "stable"]
+        python-version: ["3.10"]
```

```suggestion
        python-version: ["3.11"]
```

Not sure why we are dropping the other pythong versions but I'm 👍 with that